stripe/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -173,18 +182,53 @@ class PaymentSheetActivityTest {
         }
     }
 
+    @Test
+    fun `shows add card fragment when no payment methods available`() {
+        val viewModel = PaymentSheetViewModel(
+            publishableKey = ApiKeyFixtures.FAKE_PUBLISHABLE_KEY,
+            stripeAccountId = null,
+            stripeRepository = FakeStripeRepository(paymentIntent, listOf()),
+            paymentController = StripePaymentController(
+                ApplicationProvider.getApplicationContext(),
+                ApiKeyFixtures.FAKE_PUBLISHABLE_KEY,
+                stripeRepository
+            )
+        )
+        val scenario = activityScenario(viewModel)
+        scenario.launch(intent).onActivity { activity ->
+            // wait for bottom sheet to animate in
+            testCoroutineDispatcher.advanceTimeBy(500)
```

Comment:
Is there a way to avoid hardcoding the timing? Does `idleLooper()` alone not solve it?

Reply:
Currently, I have a manual call to `delay` in PaymentSheetActivity before showing the BottomSheet because I found that otherwise, you wouldn't be able to see it animate in. I can revisit this and see if there is a better way to do it (maybe doing it in `onResume` instead of `onCreate` will fix this?)

Reply:
Gotcha - is this value hardcoded somewhere that we can reference as a constant?

Reply:
Yeah, if we need to keep it I'll reference the constant